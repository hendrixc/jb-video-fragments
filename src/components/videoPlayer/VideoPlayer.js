import React, { Component } from 'react';

class VideoPlayer extends Component {
  render() {
    return (
      <div className="videoPlayer">
        <video
          src="/videos/video.mp4"
          controls
          // autoPlay
        >
          <track default kind="captions" />
        </video>
      </div>
    );
  }
}

export default VideoPlayer;
