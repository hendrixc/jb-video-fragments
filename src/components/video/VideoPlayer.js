import React, { Component } from 'react';
import './videoPlayer.css';
import Video from './Video';

class VideoPlayer extends Component {
  render() {
    return (
      <div className="content">
        <div style={{ display: 'flex', flex: 1, flexDirection: 'column' }}>
          <Video />
        </div>
      </div>
    );
  }
}

export default VideoPlayer;
