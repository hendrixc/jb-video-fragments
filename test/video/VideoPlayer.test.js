import React from 'react';
import renderer from 'react-test-renderer';
import VideoPlayer from '../../src/components/videoPlayer/VideoPlayer';

it('renders an home component into the VideoPlayer as snapshot', () => {
  const TextInputComponent = renderer.create(<VideoPlayer />).toJSON();
  expect(TextInputComponent).toMatchSnapshot();
});
