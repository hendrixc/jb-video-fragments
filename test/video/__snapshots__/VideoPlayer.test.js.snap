// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders an home component into the VideoPlayer as snapshot 1`] = `
<div
  className="videoPlayer"
>
  <video
    controls={true}
    src="/videos/video.mp4"
  >
    <track
      default={true}
      kind="captions"
    />
  </video>
</div>
`;
